docs/enhancement-proposals/off-ep-0001.md

```diff
@@ -0,0 +1,68 @@
+# OFF-EP 1 — Clarify that constraint distances override equilibrium bond distances
+
+**Status:** Proposed
+
+**Authors:** Matt Thompson, matt.thompson@openforcefield.org
+
+**Stakeholders:** David Mobley, John Chodera, Jeffrey Wagner, Simon Boothroyd
+
+**Acceptance criteria:** Unanimity
+
+**Created:** 2021-11-23
+
+**Discussion:** [Issue #5](https://github.com/openforcefield/standards/issues/5)
+
+**Implementation:** [PR #21](https://github.com/openforcefield/standards/pull/21)
+
+## Abstract
+
+This OFF-EP improves the documentation of the constraints section.
+
+## Motivation and Scope
+
+Most sections of the SMIRNOFF specification can be applied independently of others, but the
+`<Constraints>` section depends on the `<Bonds>` section in many implementations. Specifically, a
+pair of atoms can be subject to a harmonic bond potential via a `<Bond>` parameter and also be
+constrained via a constraint specified by a `<Constraint>` parameter. This leads to two edge cases
+that are not explicitly described in the specification.
+
+The first case is in which each section specifies a distance parameter, which are likely not
+numerically identical, and it is not clear which should be applied. The current implementation in
+the OpenFF Toolkit uses the distance specified in the `<Constraint>` record.
+
+The second case is in which a `<Constraint>` record does not specify the distance, in which case the
+`length` value of a corresponding bond parameter is used. The current implementation in the OpenFF
+Toolkit uses the distance specified in the `<Constraint>` record.
```

Comment:
This does not make sense: You are stating:
1. The `<Constraint>` record does not specify a distance
2. The OpenFF Toolkit uses the distance specified in the `<Constraint>` record

These two statements are mutually contradictory. 

Can you clarify this text to make sense? Do you mean that an exception is raised if the `<Constraint>` record does not specify the distance? Or that the distance specified in `<Bonds>` is used?

Reply:
> Do you mean that an exception is raised if the <Constraint> record does not specify the distance? Or that the distance specified in <Bonds> is used?

Oh, I thought way too hard about this. Yup -- I'm pretty sure that the second case is correct. It's just a typo.

```suggestion
Toolkit uses the distance specified in the `<Bond>` record.
```

Reply:
Yes, I think this was a typo - the result is what was intended. Thanks for catching.